daemon/info.go

```diff
@@ -26,12 +26,16 @@ import (
 // SystemInfo returns information about the host server the daemon is running on.
 func (daemon *Daemon) SystemInfo() (*types.Info, error) {
 	kernelVersion := "<unknown>"
-	if kv, err := kernel.GetKernelVersion(); err == nil {
+	if kv, err := kernel.GetKernelVersion(); err != nil {
+		logrus.Errorf("Could not get kernel version: %v", err)
```

Comment:
yup, I thinks it's more a warning than anything so I'm not sure we need to check the error at all. So I'm -1 on this.